Automattic/jetpack
# Search: Fix the December filter bug

Fixes #19819, reported by @philnick206.

#### Changes proposed in this Pull Request:
* Fixes a bug that malformed the API request when filtering for results posted/updated in December.
* Adds test coverage to the code responsible for this bug.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Add a Jetpack Search widget with a Date filter (Type: Date, Interval: Month).
* Ensure that you have a searchable post published in December.
* Open the search modal and select a December month filter.
* Ensure that the post appears as one of the search results.
